src/modules/coreHttpApi/openapi.yml

```diff
@@ -5615,6 +5614,20 @@ components:
             - $ref: "#/components/schemas/Address"
           nullable: false
           description: The address of the peer identity.
+        peerIdentity:
```

Comment:
`peerIdentity` must be added below to required. Also, to be more consistent with our code, I'd prefer it if you move this below `peerDeletionInfo`.

Reply:
It's consistent to how the object will occur so I leave it where it is.